.buildkite/pipeline.yml

```diff
@@ -5,12 +5,11 @@ env:
   # BUILDKITE_PLUGIN_DOCKER_COMPOSE_UPLOAD_CONTAINER_LOGS: "always"
   BUILDKITE_PLUGIN_DOCKER_COMPOSE_PULL_RETRIES: 5
   BUILDKITE_PLUGIN_DOCKER_COMPOSE_PUSH_RETRIES: 5
-  PLUGIN_DOCKER_COMPOSE_VERSION: "17bac3aaee1360e39381b89bb45513b11838238e"
-  PLUGIN_DOCKER_CACHE_VERSION: "f3d8feb52a25c69c75565e9f0b80375eae51850a"
+  PLUGIN_DOCKER_COMPOSE_VERSION: "e88235edd25e436a810da1ce161c04237c7a6f2c"
+  PLUGIN_DOCKER_CACHE_VERSION: "ffd18c5910ad402493e8a6614536b4e2b286d665"
 
 steps:
   - label: ":docker: Build"
-    command: ./docker/ci-prep.sh
```

Comment:
I couldn't find a file that corresponded with this? Was this just cruft?

Reply:
Yes! It's copypasta that's been throwing non-fatal errors this entire time!